chapters/interactions-nonlinear.qmd

```diff
@@ -789,6 +789,195 @@ The H-statistic and its alternatives can be an incredibly valuable tool for lear
 
 ## Discretization
 
+
+## Sidebar: The Variance-Bias Tradeoff {#sec-variance-bias}
+
+Our discussion of basis expansions presents an excellent time for a segue to discuss an essential idea in statistics and modeling: the variance-bias tradeoff. This idea will be relevant in upcoming sections for resampling and specific models. At the end of this section, we’ll also connect it to @sec-effect-encodings. 
+What does variance mean in terms of a machine learning model? In this context, it would quantify how much the fitted model changes if we slightly change the data. For example, would the curve change much if we repeat the data collection that produces the values in @sec-polynomials and fit the sample model (for a fixed sample size and polynomial degree)? We can also look at the _variance of prediction_: for a specific new data point $x_0$, how much intrinsic uncertainty is there? Similar to the discussion regarding extrapolation of global polynomials, we might want to compare how much the uncertainty changes as we move outside the training set’s range. 
```

Comment:
Some references here are to sections/equations in PR https://github.com/aml4td/website/pull/37